### PR TITLE
news

### PR DESCRIPTION
**Pull request by igorv404**

### CODE REVIEW

File: app/src/main/java/io/moiro/vision/ui/theme/Theme.kt [hunk 1]
```diff
@@ -12,7 +12,8 @@ private val LightColorPalette = lightColors(
     onPrimary = White,
     onBackground = Black,
     secondary = Grey,
-    error = Red
+    error = Red,
+    onError = LightGrey
```
Comment (DmtkD) [inline app/src/main/java/io/moiro/vision/ui/theme/Theme.kt:16]: Why do we use this fragment code?

File: app/src/main/java/io/moiro/vision/screens/news/NewsScreen.kt [hunk 2]
```diff
@@ -13,31 +14,92 @@ import androidx.compose.ui.Alignment
...
+fun NewsScreen() {
     val systemUiController = rememberSystemUiController()
     val darkTheme = isSystemInDarkTheme()
     SideEffect {
         systemUiController.setStatusBarColor(
             color = if (darkTheme) Color(0xFF1A7474) else Color(0xFFFFFFFF)
         )
     }
-    Column(
+    LazyColumn(
```
Comment (DmtkD) [inline app/src/main/java/io/moiro/vision/screens/news/NewsScreen.kt:31]: What is the difference between them?

File: app/src/main/java/io/moiro/vision/ui/theme/Theme.kt (hunk 1 above)
Comment (igorv404) [reply app/src/main/java/io/moiro/vision/ui/theme/Theme.kt:16]: to add new colours for the theme

File: app/src/main/java/io/moiro/vision/screens/news/NewsScreen.kt (hunk 2 above)
Comment (igorv404) [reply app/src/main/java/io/moiro/vision/screens/news/NewsScreen.kt:31]: lazy colum is used for scrolling

**Pull request merged**